CoinGecko.cs

```diff
@@ -16,34 +16,43 @@
 
 using System;
 using NodaTime;
-using ProtoBuf;
 using System.IO;
 using QuantConnect.Data;
 using System.Collections.Generic;
+using System.Globalization;
 
 namespace QuantConnect.DataSource
 {
     /// <summary>
-    /// Example custom data type
+    /// Universe Selection for Coin Gecko data which contains Price, Volume and Market Cap
```

Comment:
> contains Price

Price of the coin in USD or against what currency? think we should clarify it 👍 same for the universe version

> Universe Selection

This comment seems just for the universe version?